services/common/src/interfaces/projects/projectLink.interface.ts

```diff
@@ -0,0 +1,27 @@
+interface IProjectLinkContact {
+  name: string;
+}
+
+interface IProjectLinkSummary {
+  status_code: string;
```

Comment:
This one looks like it might want to be an enum instead of a string, which we probably should have done before. Do we have a finite list of project statuses?

Reply:
Oh yeah!  Love me a good enum!